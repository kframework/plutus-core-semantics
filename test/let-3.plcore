(program

(module Foo (imported) (exported () ()) (local () ())
(define result
 (let 4 four
 (let 10 ten
 (let (lam x (lam y (builtin multiplyInteger x y))) times
   [[times 4] 10]))))
)

)
