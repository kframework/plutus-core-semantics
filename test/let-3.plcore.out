<T> <k> . </k> <lenv> .Map </lenv> <genv> ( Foo . result ) |-> 40 </genv> <mod> Foo </mod> <sym> .Map </sym> <gas> 9976 </gas> </T>
